src/sss_client/common.c

```diff
@@ -919,39 +919,25 @@ int sss_pac_check_and_open(void)
     return EOK;
 }
 
+/* Non-locking version is used by single threaded 'krb5_child' */
```

Comment:
Hi,

but it is also called by the sssd_pac Kerberos plugin via `sss_pac_make_request_with_lock()`. The plugin might be loaded into threaded applications. Does this make a difference?

bye,
Sumit

Reply:
This comment was meant to explain why non-locking `sss_pac_make_request()` is exported / non-static.

Plugin uses locked `sss_pac_make_request_with_lock()`

I can remove comment if you think it's confusing.

Reply:
Updated wording.

Reply:
Thanks